util/types.ts

```diff
@@ -1,20 +1,22 @@
 export interface Project {
-  name: string,
-  description: string,
-  repo: string,
-  link: string,
-  image: string,
-  alt: string,
-  lang: Language,
+  name: string;
+  description: string;
+  repo: string;
+  link: string | null;
```

Comment:
I would consider - do you want to make this type nullable, instead of explicitly a union with `null`?

Reply:
Making this type nullable would require extra steps in our mapping function of the `getProjects` function then. Should we add in extra steps of our mapping function to make the link field nullable?

Reply:
Ah sorry, I miswrote - meant to say optional ("nullable" is not a special type in TS). It is a more generic type than `string | null`, so I think there's an argument that swings either way. If you think this is the better approach, feel free to keep it as-is.

Reply:
On second thought, I think nullable does make more sense.